src/test/java/page/GoogleMailHomePage.java

```diff
@@ -47,26 +47,30 @@ public String getHomepageUrl() {
         return HOMEPAGE_URL;
     }
 
-    public GoogleMailCreateNewEmailPage createNewEmail() {
+    public GoogleMailCreateNewEmailPage goToCreateNewEmail() {
         createNewEmailButton.click();
+        log.info("Create new email page opened");
         return new GoogleMailCreateNewEmailPage(driver);
     }
 
     public GoogleMailSentEmailsPage openSentEmailsPage() {
         sentEmailsFolderLink.click();
         wait.until(ExpectedConditions.urlMatches(sentEmailsPage.SENT_EMAILS_URL));
+        log.info("Sent emails page opened");
         return new GoogleMailSentEmailsPage(driver);
     }
 
     public GoogleMailDraftsPage openDraftsPage() {
         draftsFolderLink.click();
         wait.until(ExpectedConditions.urlMatches(draftsPage.DRAFT_EMAILS_URL));
+        log.info("Drafts page opened");
         return new GoogleMailDraftsPage(driver);
     }
 
     public void logoutGoogleAccount() {
         googleAccountInfoButton.click();
         quitGoogleAccountButton.click();
+        log.info("Logout success. Test passed\n**************");
```

Comment:
This is not a test layer. You cannot guarantee that it is the final stage of the test.